circuit_maintenance_parser/provider.py

```diff
@@ -186,6 +186,7 @@ class AWS(GenericProvider):
     """AWS provider custom class."""
 
     _processors: List[GenericProcessor] = [
+        CombinedProcessor(data_parsers=[EmailDateParser, TextParserAWS1]),
```

Comment:
have you tried to invert them? so we test first the most complete parsing, including the Subject, and only then it fails, we try the second approach.

Reply:
yep! I think we had even hinted in that direction, to ensure that the more complete parser would take precedent in it's processing.  Thanks for the reminder! I've swapped those!